Install/toolbox/scripts/depth_statistics.py

```diff
@@ -121,6 +122,22 @@ def main(in_raster=None, neighborhood_size=None,
                 utils.msg("saving mean depth to {}".format(mean_raster))
             arcpy.CopyRaster_management(mean_depth, mean_raster)
 
+            if diff_mean_set.intersection(out_stats):

```

Comment:
It looks like if a user selects `Difference to Mean` but not `Mean Depth`, the tool will not produce any results. Can we change this to calculate `Mean Depth` either way, so that `Difference to Mean` can be calculated alone?

Reply:
Yup, great catch! Let me fix this and resubmit.